Add rocksdb (#527)
VeriBlock/alt-integration-cpp: bindings/go/CMakeLists.txt

if(TESTING AND TESTING_GO)
    if(NOT WITH_C_WRAPPER)

>         message(FATAL_ERROR "you should build library with the c wrapper, -DWTIH_C_WRAPPER=ON")

Reviewer: ```suggestion
        message(FATAL_ERROR "Please build library with the c wrapper, -DWTIH_C_WRAPPER=ON")
```

Author: fixed
